oozie-to-airflow/README.md

```diff
@@ -315,3 +314,94 @@ To run all the tests in a given file call the below command:
 ```
 python -m unittest /path/to/test/file.py
 ```
+
+## System Tests
+
+Oozie to Airflow has a set of system tests that test end-2-edn functionality of conversion and execution
+of workflows using Cloud Dataproc and Cloud Composer.
+
+### System test environment
+
+Cloud Composer:
+* composer-1.5.0-airflow-1.10.1
+* python version 3 (3.6.6)
+* machine n1-standard-1
+* node count: 3
+* Additional pypi packages:
+    * sshtunnel==0.1.4
+
+Cloud Dataproc Cluster with Oozie
+* n1-standard-2, 2 vCPU, 7.50 GB memory
+* primary disk size, 50 GB
+* Image 1.3.29-debian9
+* Hadoop version
+* Init action: TODO
+
+
+### Running the tests
+
+We can run examples defined in examples folder to run system test. The system tests use a real
+dataproc cluster, composer and oozie (in the future) to run the tests automatically.
+
+Each test consists of several phases:
+
+* convert - converts workflow from the example folder to DAG
```

Comment:
```suggestion
* convert - converts workflow from the example folder to an Airflow DAG
```
In Oozie they also sometimes call workflows DAGs, so maybe it's better to be specific.